tsconfig.json

```diff
@@ -23,6 +30,6 @@
     },
     "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
```

Comment:
could you tell me why was this needed?

Reply:
Honestly, I do not remember adding that. Maybe it was added during one of my yarn commands.

I looked into it more, and found that I built the app before pushing my commits. `yarn build` gave this message:
![image](https://github.com/Free-Code-Camp-Seoul/fcc-seoul-website/assets/44438849/39fe1161-d113-48d3-9f00-403ae97f8dde)

Reply:
I guess this is not a best practice since it is optimizing for production and not what we use for development.

Reply:
oh that's interesting, I guess we can keep it then